core/src/main/resources/jenkins/diagnostics/ControllerExecutorsNoAgents/message.jelly

```diff
@@ -0,0 +1,34 @@
+<!--
+The MIT License
+
+Copyright (c) 2021, CloudBees, Inc.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <div class="alert alert-warning">
+    <form method="post" action="${rootURL}/computer/(master)/configure" name="${it.id}">
```

Comment:
This form target makes no sense:

* "Manage" should take me somewhere where I can configure agents or clouds. Neither is the case with this target. I expect we'll need two buttons, "Set up Agent" and "Set up Cloud" linking to `/computer/new` and `/configureClouds/` respectively (or just one link to `/computer/` but that seems less user friendly).
* With this form target, the dismiss button doesn't work at all.

There's a reason all of these monitors use a web method that barely does anything in almost all cases.